README.md

```diff
@@ -15,30 +14,7 @@
    +  Create documents
    +  Delete documents, users
    +  Update documents, users
-
-## Endpoints
```

Comment:
If I wanted to contribute to your project, I'd be hurt if I didn't see it in your `README`. The `README` is like a developers go-to source for information about a project. The more descriptive, the better.